src/main/java/org/jalasoft/moi/model/Params.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package org.jalasoft.moi.model;
+
+import java.nio.file.Path;
+
+/**
+ * Parms class contains information from client about what language and root are need
```

Comment:
Please check the description suits the class name

Reply:
ok